cmd/api-linter/integration_test.go

```diff
@@ -96,6 +96,19 @@ func TestRules_DisabledByFileComments(t *testing.T) {
 	}
 }
 
+func TestRules_DisabledByFileCommentsAndIgnoreCommentDisablesTrue(t *testing.T) {
+	for _, test := range testCases {
+		t.Run(test.testName, func(t *testing.T) {
+			disableInFile := fmt.Sprintf("// (-- api-linter: %s=disabled --)", test.rule)
+			proto := disableInFile + "\n" + test.proto
+			_, result := runLinterWithFailureStatus(t, proto, "", []string{"--ignore-comment-disables"})
+			if !strings.Contains(result, test.rule) {
+				t.Errorf("Rule %q should be enabled despite a disable by file comments", test.rule)
```

Comment:
Errors should neither start with a capital letter nor end in a period. IOW, downcase, "Rule".

Reply:
That's for the `err` type, correct? There are examples even in internal go styleguides that have capital letter:

```
func mustAddGameAssets(t *testing.T, dir string) {
    t.Helper()
    if err := os.WriteFile(path.Join(dir, "pak0.pak"), pak0, 0644); err != nil {
        t.Fatalf("Setup failed: could not write pak0 asset: %v", err)
    }
    if err := os.WriteFile(path.Join(dir, "pak1.pak"), pak1, 0644); err != nil {
        t.Fatalf("Setup failed: could not write pak1 asset: %v", err)
    }
}
```

Reply:
I don't have a strong opinion so I'm fine with switching, but this was just the precedent of the whole file. I'll have to change them all.

Reply:
Personally I think `t.Errorf` is not the same as constructing an `error` message returned to a user, so I use whatever casing :)